rfq/order.go

```diff
@@ -53,12 +54,23 @@ func parseHtlcCustomRecords(customRecords map[uint64][]byte) (*rfqmsg.Htlc,
 // SerialisedScid is a serialised short channel id (SCID).
 type SerialisedScid = rfqmsg.SerialisedScid
 
+// AssetSpecifierChecker is an interface that contains methods for checking
+// certain properties related to asset specifiers.
+type AssetSpecifierChecker interface {
```

Comment:
Same here re duplication, I think we only need one instance, unless some of these are in place to avoid import cycles (doesn't appear so).